spec/player_symbols_spec.rb

```diff
@@ -18,8 +18,8 @@
     expect(PlayerSymbols::opponent(PlayerSymbols::O)).to be :X
   end
 
-  it "all player symbols" do
-    expect(PlayerSymbols::all).to eq [:X, :O]
+  it "all player symbols as strings" do
+    expect(PlayerSymbols::all).to eq [:X.to_s, :O.to_s]
```

Comment:
Prefer being specific in the test, so just ["X", "O"] instead of duplicating the to_s in both production and test code.